queries.sql

```diff
@@ -1,3 +1,27 @@
-/*Queries that provide answers to the questions from all projects.*/
+/*Queries that provide answers to the questions FROM all projects.*/
+
+-- Find all animals whose names ends in "mon".
+
+SELECT * from animals where names LIKE '%mon';
+
+-- List the names of all animals born between 2016 and 2019.
+SELECT names from animals where date_of_birth  BETWEEN '2016-01-01' AND '2019-12-31';
+
```

Comment:
- [ ] Kindly make sure that you add screenshots for the one you have missed :
1) SELECT * from animals where names LIKE '%mon';
2) SELECT names from animals where date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';